sdk/src/types/block/semantic.rs

```diff
@@ -112,7 +113,7 @@ pub struct ValidationContext<'a> {
     ///
     pub unlocks: &'a Unlocks,
     ///
-    pub milestone_timestamp: u32,
+    pub slot_index: SlotIndex,
```

Comment:
seems like we should either silence clippy appropriatedly (with an attribute) or document this struct (also separate PR)